docs/static_site/src/_includes/get_started/linux/scala/gpu.md

```diff
@@ -1,6 +1,6 @@
 You can use the Maven packages defined in the following dependency to include MXNet in
 your Scala
-project. Please refer to the <a href="scala_setup.html">MXNet-Scala setup guide</a> for
+project. Please refer to the <a href="/get_started/scala_setup.html">MXNet-Scala setup guide</a> for
```

Comment:
Why is this different from the cpu.md?

Reply:
The current Scala link is broken. I modify the path to correct it.
The only difference between cpu.md and gpu.md is the dependency tag, which makes them not easy to merge.